src/main/scala/cromwell/CommandLineParser.scala

```diff
@@ -64,7 +64,7 @@ object CommandLineParser extends App {
         children(
           arg[String]("workflow-source").text("Workflow source file.").required().
             action((s, c) => c.copy(workflowSource = Option(DefaultPathBuilder.get(s)))),
-          opt[String]('i', "workflow-root").text("Workflow root.").
+          opt[String]("workflow-root").text("Workflow root.").
```

Comment:
TOL even if we did want a one char mnemonic I'm not sure why that one 😕

Reply:
I assumed it was copied from `opt[String]('i', "inputs")` below.

Reply:
:man_facepalming: